mypy.ini

```diff
@@ -0,0 +1,16 @@
+[mypy]
+
+strict = True
+
+; We intend to type check all modules.
+; However, while we are in the process of adding type hints, we want to
+; prevent new errors from being introduced.
+; Therefore, we have a strict mypy config, but we temporarily ignore errors
+; in existing modules which have not yet been fully type annotated.
+;
+; The goal is to remove "ignore_errors" from this file once all modules
+; have been fully type annotated.
+
+[mypy-sybil.document,sybil.evaluators.skip,sybil.integration.pytest,sybil.parsers.abstract.codeblock,sybil.parsers.abstract.doctest,sybil.parsers.myst.codeblock,sybil.parsers.myst.doctest,sybil.parsers.rest.capture,sybil.parsers.rest.doctest,sybil.sybil]
```

Comment:
An alternative to this long line is to use multiple sections.
I have no strong preference as each option has its trade-off.
I expect to shorten this line by removing type errors.

Reply:
I think it's better to not have this. I'd like this branch to be one that fails until it's rebased on `master`, once that branch is ready to have this one landed.

This branch, however, should have any global config we need to ignore mypy complaints that will never be addressed.

Reply:
Ok